exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,5 @@
-const name = " Daniel  ";
+const name = " Daniel ";
+nameNew = name.trim();
```

Comment:
`var` or `let` is missing before `nameNew`! This way it is created as a global variable, which is something that we'd like to avoid.

Reply:
I haven't realised it. I solved it. Thank you!